cli/cli.go

```diff
@@ -26,17 +26,27 @@ func Run() {
 			Usage:  "debug mode",
 			EnvVar: "DEBUG",
 		},
-
 		cli.StringFlag{
 			Name:  "log-level, l",
 			Value: "info",
 			Usage: fmt.Sprintf("Log level (options: debug, info, warn, error, fatal, panic)"),
 		},
+		cli.StringFlag{
+			Name:  "log-file",
+			Usage: "Logging the output to a file instead of stdout",
+		},
 	}
-
 	// logs
 	app.Before = func(c *cli.Context) error {
-		log.SetOutput(os.Stderr)
+		if c.IsSet("log-file") {
+			logDir := c.String("log-file")
+			fmt.Print(logDir)
+			file, err := os.OpenFile(logDir, os.O_RDWR, 0777)
+			fmt.Print(err)
```

Comment:
@CiaranCostello Imho, falling back to the default way to do logging should be the behavior (but we should probably log it as a **Warning**: `Error creating log file: falling back to default method`).

It's not necessary to fail hard on this one as this will cause much confusion for users running the Swarm Image in a container and don't have a direct access to logs.